### PR TITLE
docs: 修复中文文档没有与英文文档同步的问题

### LINKED ISSUE #33867: 中英文语言下设计模式概览内容不一致
- [ ] I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.

### Reproduction link
[https://ant.design/docs/spec/overview-cn](https://ant.design/docs/spec/overview-cn)

### Steps to reproduce
切换中英文

### What is expected?
内容一致

### What is actually happening?
不一致

| Environment | Info |
|---|---|
| antd | undefined |
| React | react |
| System | macOS  version 11.6 |
| Browser | google |

### PR DESCRIPTION
**Pull request by c0dedance**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [x] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [x] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 🔗 Related issue link

close #33867

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |           |
| 🇨🇳 Chinese |  update  |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: docs/spec/overview.zh-CN.md [hunk 1]
```diff
@@ -6,16 +6,27 @@ title: 概览
...
+- 组件 - 基本组件：系统的最基本元素，例如按钮和寻呼机。- 业务组件/模块：块级示例，通常由多个组件组成，例如 PageHeader 通用标题。
+- 一般概念：保证 ETC 系统化的一些约定，例如排版、字体和文案。
+
+## 资源
+
+我们与工程师合作，将设计模式转化为可重用的代码，最大限度地提高您的生产力和沟通效率。
+
+- [Ant Design Pro](https://pro.ant.design/)：具有 20 多个模板和 10 多个业务组件的开箱即用解决方案
+- [Ant Design Components](https://ant.design/docs/react/introduce)：Ant Design 的 React UI 库是一个拥有 50 多个基础组件的全局组件库
+- [Ant 设计库](http://library.ant.design/)：代码中包含 Axure 资源包，使您的原型看起来像一个视觉草稿，包括模板、组件等。
```
Comment (afc163) [inline docs/spec/overview.zh-CN.md:32]:
```suggestion
- [Axure 设计库](http://library.ant.design/)：代码中包含 Axure 资源包，使您的原型看起来像一个视觉草稿，包括模板、组件等。
```

File: docs/spec/overview.zh-CN.md [hunk 2]
```diff
@@ -6,16 +6,27 @@ title: 概览
...
+- 模板：一个页面级示例，启发用户如何在系统中构建典型页面，例如详细信息页面。
+- 组件 - 基本组件：系统的最基本元素，例如按钮和寻呼机。- 业务组件/模块：块级示例，通常由多个组件组成，例如 PageHeader 通用标题。
+- 一般概念：保证 ETC 系统化的一些约定，例如排版、字体和文案。
+
+## 资源
+
+我们与工程师合作，将设计模式转化为可重用的代码，最大限度地提高您的生产力和沟通效率。
+
+- [Ant Design Pro](https://pro.ant.design/)：具有 20 多个模板和 10 多个业务组件的开箱即用解决方案
+- [Ant Design Components](https://ant.design/docs/react/introduce)：Ant Design 的 React UI 库是一个拥有 50 多个基础组件的全局组件库
```
Comment (afc163) [inline docs/spec/overview.zh-CN.md:31]:
```suggestion
- [React 官方实现](https://ant.design/components/overview-cn/)：Ant Design 的 React UI 库拥有 60 多个基础组件。
```

File: docs/spec/overview.zh-CN.md [hunk 3]
```diff
@@ -6,16 +6,27 @@ title: 概览
...
-通过规范常见的互动行为，包括导航、数据录入、数据展示、反馈、操作、帮助等，呈现组件元素的标准用法和弹性空间，并了解如何将他们组织在一起来创建良好的体验。
+![结构图](https://gw.alipayobjects.com/zos/rmsportal/NyWYOFQxJYElAwtUfSdv.png)
 
-### 模板文档
+完整的设计模式将包括模板、组件 (ETC) 和通用概念的示例：
 
-基于蚂蚁中台的设计经验，我们总结了表单、列表、数据可视化、详情页、工作台、异常、结果、编辑器几类页面的典型解决方案，介绍每类页面的设计目标、设计技巧以及典型页面内容等，帮助广大设计者可以快速开始构建界面。你还可以在 [Ant Design Pro](https://pro.ant.design/index-cn) 里在线预览这些典型页面。
+- 功能示例：由多个模板组成，以启发用户如何使用和构建一个通用功能。
+- 模板：一个页面级示例，启发用户如何在系统中构建典型页面，例如详细信息页面。
+- 组件 - 基本组件：系统的最基本元素，例如按钮和寻呼机。- 业务组件/模块：块级示例，通常由多个组件组成，例如 PageHeader 通用标题。
```
Comment (afc163) [inline docs/spec/overview.zh-CN.md:23]:
```suggestion
- 组件
   - 基本组件：系统的最基本元素，例如按钮和分页器。
   - 业务组件/模块：块级示例，通常由多个组件组成，例如 PageHeader 通用标题。
```

File: docs/spec/overview.zh-CN.md [hunk 4]
```diff
@@ -6,16 +6,27 @@ title: 概览
 
 > 这是一份动态更新的设计文档，你的阅读和反馈是我们前进的动力。
 
-秉承 [Ant Design 设计价值观](/docs/spec/values)，针对企业级产品反复出现的设计问题，设计模式给出一般性的解决方案，实现跨应用交互一致性的有效融合。依照设计模式使用 [antd 组件](/docs/react/introduce)设计界面，既减少了不必要的认知成本，又能够提升交互确定性和设计的效率。考虑到需要适应各种独特的业务场景，模式的规则具有一定的灵活性，万变不离其宗，通过了解设计模式的构建逻辑，可以衍生出更具场景特性的解决方案。设计模式包含以下内容：
+在企业级业务中使用设计模式，可以显着增加研发团队的确定性，节省不必要的设计，保持系统一致性，让“设计师”专注于创意最需要的地方。
```
Comment (afc163) [inline docs/spec/overview.zh-CN.md:9]:
```suggestion
在企业级业务中使用设计模式，可以显着增加研发团队的确定性，节省不必要的设计，保持系统一致性，让『设计师』专注于创意最需要的地方。
```

**Comment by c0dedance** [comment]
我觉得你贡献的更多哈哈哈

**Pull request merged**